website/react-magma-docs/src/pages/api/modal.mdx

```diff
@@ -197,6 +221,56 @@ export function Example() {
           <Button onClick={() => setShowModalHeader(false)}>OK</Button>
         </ButtonGroup>
       </Modal>
+
+      <Modal
+        onClose={() => {
+          setShowModalHeaderRef(false);
+          headerRefButtonRef.current.focus();
+        }}
+        isOpen={showModalHeaderRef}
+        headerRef={handleGetHeaderRef}
+        style={{ padding: '20px' }}
+      >
+        <Flex
+          behavior={FlexBehavior.container}
+          direction={FlexDirection.column}
+          alignItems={FlexAlignItems.center}
+          justify={FlexJustify.center}
+          spacing={2}
+          wrap={FlexWrap.nowrap}
+        >
+          <CheckIcon
+            style={{
+              background: magma.colors.success,
+              color: magma.colors.neutral100,
+              width: '36px',
+              height: '36px',
+              padding: '8px',
+              borderRadius: '50%',
+            }}
+          />
+          <h2 ref={customHeadingRef} tabIndex={-1}>
+            Add Links to Your Content
+          </h2>
+          <Paragraph>
+            Your course is now integrated. Select specific activities to provide
+            students with direct links. Or, add just one link to your entire
+            course. You can return to select more content later.
+          </Paragraph>
```

Comment:
Maybe we make this even more generic? It doesn't have to be course/activity related.